src/ray/core_worker/core_worker.cc

```diff
@@ -532,12 +532,8 @@ void CoreWorker::Shutdown() {
 }
 
 void CoreWorker::Disconnect() {
-  io_service_.stop();
```

Comment:
If we don't stop the io_service here, it will be stopped in the destructor, right?